fedimint-client/src/lib.rs

```diff
@@ -722,7 +722,7 @@ pub struct ClientModuleInstance<'m, M: ClientModule> {
     /// Module-specific API
     pub api: DynModuleApi,
 
-    module: &'m M,
+    pub module: &'m M,
```

Comment:
This type defers to `M` (right below), so I don't think we need this to be public.

Reply:
defer doesn't have same lifetime :/

if I remove .module in wasm-client/lib.rs, it fails due to lifetime error. 
I can make it a getter.